src/Components/PresentationalComponents/CVEDetailsPageDescription/CVEDetailsPageDescription.js

```diff
@@ -36,7 +52,8 @@ const CVEDetailsPageDescription = ({ cveAttributes, intl }) => {
                             style={{ height: '132px', width: '100%' }}
                         >
                             <TextContent style={{ textAlign: 'justify' }}>
-                                <Text component={TextVariants.p}>{description}</Text>
+                                <Text component={TextVariants.p}>{description ?
+                                    description : intl.formatMessage(messages.noMetadataDescription)}</Text>
```

Comment:
This can be simplified to `{description || intl.formatMessage(messages.noMetadataDescription)}`